calico-enterprise_versioned_docs/version-3.20-2/release-notes/index.mdx

```diff
@@ -140,6 +140,10 @@ $[prodname] 3.20 contains breaking changes for installations that use the Calico
   If you're using the Calico API server, you must restart any controllers, including `kube-controller-manager`, that manage these resources after the upgrade.
   This change addresses an issue where duplicate UIDs on different API resources could disrupt Kubernetes garbage collection.
 
+* ***Important change:*** In Calico Enterprise 3.20.0-2.0, the order of the default tier is changed to 1,000,000. This changes tiers evaluation if there is another tier with higher order value.
+  It's important to verify the existence of tiers with higher order numbers before upgrading to Calico Enterprise 3.20.0-2.0 or newer releases.
+  In case such a tier exists, the order number must be changed to a lower value than 1,000,000 to guarantee its evaluation before the default tier.
```

Comment:
***Breaking change:*** Previously, the default tier had no order and was always evaluated last. Starting with Calico Enterprise 3.20.0-2.0, the default tier now has an order of 1,000,000. When upgrading, you must ensure that existing tiers have a lower order or else policy decisions may be affected.